spec/exchanges/btse_futures/integration/market_spec.rb

```diff
@@ -80,11 +80,11 @@
       expect(contract_stat.base).to eq 'BTC'
       expect(contract_stat.target).to eq 'USD'
       expect(contract_stat.market).to eq 'btse_futures'
-      expect(contract_stat.index).to be nil
+      expect(contract_stat.index).to be_a Numeric
       expect(contract_stat.open_interest).to be_a Numeric
       expect(contract_stat.timestamp).to be nil
-      expect(contract_stat.index_identifier).to eq 'BtseFutures-BTC'
```

Comment:
@tmlee how do you get this format of naming for `index_name` and `index_identifier`

Reply:
That is a placeholder for exchange that do not support index prior.
Since now this one has it, we should use its `id`